### ISSUE TITLE
Terminal doesn't work in VSCode Insiders

### ISSUE THREAD

**Issue by usernamehw** [microsoft/vscode#105431]
Issue Type: <b>Bug</b>

This issue started a year ago ... maybe even before that. The error in devtools says that it cannot connect to winpty.

I believe that it is somehow connected with Windows version being wrong (when you invoke `Help:About`: (the first one is stable)

```diff
+ OS: Windows_NT x64 10.0.18363
- OS: Windows_NT x64 6.2.9200
```

VS Code version: Code - Insiders 1.49.0-insider (7df29b1236ecf8a21af9fe19a52923f6d69e0577, 2020-08-26T07:58:02.868Z)
OS version: Windows_NT x64 6.2.9200

**Comment by usernamehw** [comment]
https://github.com/Microsoft/vscode/issues?q=is%3Aissue+label%3Aintegrated-terminal+9200+is%3Aclosed

**Comment by usernamehw** [comment]
Both values `terminal.integrated.windowsEnableConpty` **true** and **false** do not work.

**Comment by usernamehw** [comment]
I found this issue: https://github.com/microsoft/vscode-remote-release/issues/3243 that says that vscode Insiders is running in compatibility mode with Windows 8

![Screenshot (321)](https://user-images.githubusercontent.com/9638156/91323207-4fb62000-e7c9-11ea-8203-b07fb14c6d84.png)

After unchecking compatibility mode - it starts working.

**Comment by usernamehw** [comment]
@jiankafei @Sriram23S @BarunBapari @ajithsubash @itkhanz @arielsumeruk @Unrelenting2

Hello. I found your issues are similar to mine. If:

- Operating system is Windows 10
- The `Help:About` page shows wrong OS version (that of Windows 8 `6.2.9200`)
- The Integrated terminal doesn't work

Then you probably have the same issue. What might help is to uncheck compatibility mode checkbox in app shortcut properties. https://github.com/microsoft/vscode/issues/105431#issuecomment-680949825

**Comment by BarunBapari** [comment]
> I found this issue: [microsoft/vscode-remote-release#3243](https://github.com/microsoft/vscode-remote-release/issues/3243) that says that wrong version of vscode might be because it's running in compatibility mode.
> 
> ![Screenshot (321)](https://user-images.githubusercontent.com/9638156/91323207-4fb62000-e7c9-11ea-8203-b07fb14c6d84.png)
> 
> After unchecking compatibility mode - the terminal starts working.

@usernamehw  you are boss!!
i fixed my problem..Thank you.

**Comment by Tyriar** [comment]
Yes compatibility mode breaks the terminal: https://code.visualstudio.com/docs/supporting/troubleshoot-terminal-launch#_the-terminal-exited-with-code-1-on-windows-10

**Comment by Tyriar** [comment]
This is what it looks like:

![image](https://user-images.githubusercontent.com/2193314/92183957-aa243180-ee04-11ea-82b4-afd21d3400e0.png)

Maybe there's something we can do to better detect this then not even bother to launch the process and show a useful notification.

I'm glad I now know what Windows 6.2.9200 means on Windows 10 (windows 8 compat):

![image](https://user-images.githubusercontent.com/2193314/92184007-ca53f080-ee04-11ea-8011-3816dcf40634.png)

**Comment by roblourens** [comment]
It seems like there were no changes here right? Getting it out of the verification query
